src/custom-operations/parse-schema.ts

```diff
@@ -62,6 +67,6 @@ export async function parseSchemasV2(parser: Parser, detailed: DetailedAsyncAPI)
 }
 
 async function parseSchemaV2(parser: Parser, item: ToParseItem) {
-  item.value[xParserOriginalSchemaFormat] = item.input.schemaFormat;
```

Comment:
I see you are removing it but I can't see where we add `xParserOriginalSchemaFormat` value back.

Reply:
We don't remove schemaFormat from message in custom parsers so we don't need reset to extension. Is it clear?

Reply:
Then we need to modify the README as well

Reply:
Ok, I will change it! :)